src/chunkserver/datastore/file_pool.cpp

```diff
@@ -217,18 +218,30 @@ FilePool::FilePool(std::shared_ptr<LocalFileSystem> fsptr)
 
 bool FilePool::Initialize(const FilePoolOptions &cfopt) {
     poolOpt_ = cfopt;
+    //LOG(ERROR) << "filePoolDir:" << poolOpt_.filePoolDir;
+    //LOG(ERROR) << "fileSize:" << poolOpt_.fileSize;
+    //LOG(ERROR) << "preAllocateNum:" << poolOpt_.preAllocateNum;
     if (poolOpt_.getFileFromPool) {
-        if (!CheckValid()) {
-            LOG(ERROR) << "check valid failed!";
-            return false;
+        currentdir_ = poolOpt_.filePoolDir;
+        currentState_.chunkSize = poolOpt_.fileSize;
+        currentState_.metaPageSize = poolOpt_.metaPageSize;
+        if (!fsptr_->DirExists(currentdir_.c_str())) {
```

Comment:
First check whether the existing chunkfile is legal, and then format the newly added chunkfile in the sub-thread